source/OpenBVE/UserInterface/formMain.Packages.cs

```diff
@@ -765,7 +765,7 @@ private void buttonCreatePackage_Click(object sender, EventArgs e)
 				if (!Program.CurrentlyRunningOnMono)
 				{
 					//Mono doesn't support System.Security.AccessControl, so this doesn't work....
-					System.Security.AccessControl.DirectorySecurity ds = Directory.GetAccessControl(directory);
+					// System.Security.AccessControl.DirectorySecurity ds = Directory.GetAccessControl(directory);
```

Comment:
Would it work if we called the function but just didn't assign it to a variable?